ogame/__init__.py

```diff
@@ -894,7 +916,7 @@ def send_fleet(
 
         response = self.session.post(
             url=self.index_php + 'page=ingame&component=fleetdispatch'
-                '&action=sendFleet&ajax=1&asJson=1',
+                                 '&action=sendFleet&ajax=1&asJson=1',
```

Comment:
revert autostyling of your IDE

Reply:
Oh sorry, it's Automatic...
I'll revert.